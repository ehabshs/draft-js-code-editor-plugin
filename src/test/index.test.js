// @flow
import createCodeEditorPlugin from '../';

it('should return an object', () => {
  expect(createCodeEditorPlugin()).toBeInstanceOf(Object);;
});

it('should override some hooks', () => {
  expect(createCodeEditorPlugin()).toMatchSnapshot();
});
