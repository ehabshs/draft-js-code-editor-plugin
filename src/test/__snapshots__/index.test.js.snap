// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should override some hooks 1`] = `
Object {
  "handleKeyCommand": [Function],
  "handleReturn": [Function],
  "keyBindingFn": [Function],
  "onTab": [Function],
}
`;
